DB/Item.lua

```diff
@@ -150,4 +150,17 @@ function Item:IsMount(entry)
 	return entry.type == CONSTANTS.ITEM_TYPES.MOUNT
 end
 
+function Item:HasZone(entry)
+	if entry.coords then
+		for _, value in pairs(entry.coords) do
+			if type(value) == "table" and value.m ~= nil then
+				if tonumber(value.m) then
```

Comment:
Should this check be part of the ``HasZone()`` logic? Type validation seems like it would be separate from checking the existence of required fields, though we haven't streamlined the validation enough yet to say for sure...

Reply:
Right. I forgot to take that out. Copy pasted that from an early protoype. Removed type validation in d872cdd488ff00c51da27fd74d72b4a4750ad488